### DISCUSSION TITLE
[14 settembre 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#260] [Esami]
Analizzare la rete sequenziale mostrata in figura. Stendere la tavola degli stati futuri e di uscita e disegnare l’automa (il diagramma di transizione degli stati).

In seguito, disegnare l’automa di Moore equivalente.

<div align="center">

![image](https://github.com/user-attachments/assets/e42aeeed-31f3-4c0a-bbea-2b547c5d43a8)

</div>

<details closed>
    <summary> soluzione A-L </summary>

<div align="center">

![image](https://github.com/user-attachments/assets/85ba1d77-3e05-4327-9283-94ea0e9c6d41)
![image](https://github.com/user-attachments/assets/b6669cfe-c812-464c-ad26-36a9b2a37501)

</div>
</details>

**Comment by Nw4pS** [comment]
@FeddyLix17 perché non si è preso in considerazione il clock? non essendo tutto collegato potrebbe essere che il primo e il secondo flip flop vengono attivati e il terzo no, cambiano lo stato del circuito

**Reply by FeddyLix17** [reply]
anche se non direttamente disegnato è come se i 3 flip flop fossero collegati allo stesso clock.
